.github/workflows/claude-dispatch.yml

```
@@ -0,0 +1,49 @@
+
+# IMPORTANT: Do not modify this `on` section!
+on:
+  repository_dispatch:
+    types: [claude-dispatch]
+
+jobs:
+  claude-dispatch:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)